sphinx_autobuild/build.py

```diff
@@ -28,9 +29,12 @@ def __call__(self, *, rebuild: bool = True):
             )
             raise
 
-        show(command=["sphinx-build"] + self.sphinx_args)
+        show(command=[sys.executable, "-m", "sphinx.cmd.build"] + self.sphinx_args)
```

Comment:
Why not ``-m sphinx``?

Reply:
ooh, you're right, that [does the same thing](https://github.com/sphinx-doc/sphinx/blob/71b1a06024268e39f59e3772e2420555cb9bf174/sphinx/__main__.py). I've updated the PR :)